### PR TITLE
DeviceActor: Use whenConversationExist instead of withConv or getConv

### PR DESCRIPTION
**Pull request by xuanzhaopeng**
The reason is : withConv and getConv doesn't wait until the conversation exist. If I do SendFile after Login immediately,  and if I do getConv or withConv, the actor will be failed.

### CODE REVIEW

File: actors/base/src/main/scala/com/waz/provision/DeviceActor.scala [hunk 1]
```diff
@@ -242,16 +242,17 @@ class DeviceActor(val deviceName: String,
...
-        conv <- getConv(rConvId)
-        idx <- zmessaging.messagesStorage.msgsIndex(conv.id)
-        cursor <- idx.loadCursor
-      } yield {
-
-        ConvMessages(Array.tabulate(cursor.size) { i =>
-          val m = cursor(i)
-          MessageInfo(m.message.id, m.message.msgType, m.message.time)
-        })
+      whenConversationExistsFuture(rConvId) { conv =>
```
Comment (mykola-mokhnach) [inline actors/base/src/main/scala/com/waz/provision/DeviceActor.scala]: does this future have a timeout?

Comment (xuanzhaopeng) [inline actors/base/src/main/scala/com/waz/provision/DeviceActor.scala]: Hi, just with help from Dean, we add 30 seconds as default timeout

Comment (mykola-mokhnach) [inline actors/base/src/main/scala/com/waz/provision/DeviceActor.scala]: Set it to 120  seconds- we have some very large conversations

Comment (xuanzhaopeng) [inline actors/base/src/main/scala/com/waz/provision/DeviceActor.scala]: Done 120 as default

**Pull request merged**